css/main.css

```diff
@@ -1,468 +1,481 @@
+footer {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+}
 
+body {
+    font-family: 'Roboto Mono', monospace;
+    ,
+    sans-serif;
+    margin: 0;
+    padding-left: 10%;
+    padding-right: 10%;
+}
 
-    footer {
-        position: fixed;
-        left: 0;
-        bottom: 0;
-        width: 100%;
-    }
-
-    body {
-        font-family: 'Roboto Mono', monospace;, sans-serif;
-        margin: 0;
-        padding-left: 10%;
-        padding-right: 10%;
-    }
-
-    p {
-        font-size: 0.7em;
-        line-height: 1.6em;
-        color: #707070;
+p {
+    font-size: 0.7em;
+    line-height: 1.6em;
+    color: #707070;
 
-    }
+}
 
-    header {
-        position: absolute;
-        top: 2%;
-        right: 5%;
-    }
+header {
+    position: absolute;
+    top: 2%;
+    right: 5%;
+}
 
-    img {
-        padding-top: 1em;
-    }
+img {
+    padding-top: 1em;
+}
+
+.hide-mobile {
+    display: none;
+}
+
+.underline {
+    border-bottom: 2px solid #5F6ECE;
+    padding-bottom: 0;
+
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+
+
+.container {
+    text-align: center;
+    padding: .8em;
+}
+
+
+.head-tag-div>p {
+    display: inline-block;
+    text-align: left;
+    margin-bottom: 0;
+}
+
+
+.head-tag {
+    font-size: 1.7em;
+    margin: 0;
+}
+
+.emoji {
+    font-size: 1.3em;
+}
+
+.sub-head-tag {
+    font-size: 1em;
+    margin: 0;
+}
+
+.second-sub-tag {
+    color: black;
+    font-weight: bold;
+}
+
+.white-space {
+    margin-top: 150%;
+}
+
+
+.hero {
+    display: inline-block;
+    text-align: left;
+
+}
+
+.about {
+    display: inline-block;
+    text-align: left;
+
+}
+
+.projects {
+    display: inline-block;
+    text-align: left;
+}
+
+.project-name {
+    color: black;
+    font-weight: bold;
+    font-size: 1em;
+}
+
+.technologies-used>li {
+    padding-left: 0.3em;
+
+}
+
+.skill-colon {
+    display: inline-block;
+    margin-left: 1em;
+}
+
+.not-initial-colon {
+    margin-left: 2em;
+}
+
+.button {
+    background-color: white;
+    /* Green */
+    border: 2px solid #5F6ECE;
+    color: black;
+    padding: 8px 25px;
+    text-align: center;
+    font-family: 'Roboto Mono', monospace, sans-serif;
+    font-size: .8em;
+    margin: 15px 2px;
+    -webkit-transition-duration: 0.4s;
+    text-decoration: none;
+    /* Safari */
+    transition-duration: 0.4s;
+    cursor: pointer;
+}
+
+.button:hover {
+    background-color: #5F6ECE;
+    color: white;
+    cursor: pointer;
+}
```

Comment:
Use .button class instead of directly styling buttons.
Removes the need for onclick.